posthog-core/src/index.ts

```diff
@@ -709,10 +760,16 @@ export abstract class PostHogCore extends PostHogCoreStateless {
   }
 
   resetSessionId(): void {
```

Comment:
Should it be async as well? since it needs to update the storage.

Reply:
Not sure if you followed what the `wrap` code does, but essentially nothing is async in this function.

Reply:
It's just delayed execution _if_ necessary. If the SDK is initialised that function is entirely synchronous.